lang/cabal/Portfile

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                cabal
+version             3.0.0.0
+categories          lang haskell
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         Haskell Cabal: Common Architecture for Building Applications and Libraries
+long_description    \
+    Cabal is a system for building and packaging Haskell libraries and \
+    programs. It defines a common interface for package authors and \
+    distributors to easily build their applications in a portable \
+    way. Cabal is part of a larger infrastructure for distributing, \
+    organizing, and cataloging Haskell libraries and programs.
+
+homepage            https://www.haskell.org/cabal
+
+# Bootstrapped cabal is the default build
+variant prebuilt \
+	    description {Do not bootstrap cabal; install the pre-built binary.} {}
+
+# use_xz              yes
+
+master_sites        https://downloads.haskell.org/~${name}/${name}-install-latest/
+distfiles           ${name}-install-${version}-x86_64-apple-darwin17.7.0.tar.xz
+distname            [join [lrange [split [lindex ${distfiles} end] .] 0 end-2] .]
```

Comment:
I believe that this is there wrong way. `distfiles` is automatically generated as `${distname}${extract.suffix}` and should never be manually set, especially using `distfiles`. Set the `distname` and `extract.suffix` directly instead. 

```
distname           ${name}-install-${version}-x86_64-apple-darwin17.7.0 
extract.suffix    tar.xz

```

Reply:
Tanks! Fixed.